src/tests/libecalc/integration/snapshots/test_all_consumer_with_time_slots_models/test_all_consumer_with_time_slots_models_results/all_consumer_with_time_slots_models_v3.json

```diff
@@ -630,8 +630,8 @@
                                 ]
                             },
                             "is_valid": [
-                                true,
-                                true
+                                false,
```

Comment:
Here the stage results appear as is_valid == False when the stage does not meet the requested pressure. The stage used to be valid if it was within capacity. Now that has changed.